pages/index.vue

```diff
@@ -21,14 +21,14 @@
 import { DHero, DButton } from '@brown-ccv/disco-vue-components';
 
 export default {
-  components: { DHero, DButton },
-  async asyncData({ $content }) {
-    const index = await $content('home/index').fetch();
+  components: { DHero, DButton }
+  //   async asyncData({ $content }) {
```

Comment:
why is this all commented out?

Reply:
The index is dynamic... and at the moment we are no longer copying the `content` to the `functions` because pages the use the content are all statically compiled. I left it here, because we need to discuss the landing page behavior